lib/agent0/interactive_fuzz/fuzz_long_short_maturity_values.py

```diff
@@ -0,0 +1,147 @@
+"""Script to verify that the state of pool reserves is invariant to the order in which positions are closed."""
+# %%
+from __future__ import annotations
+
+import numpy as np
+from fixedpointmath import FixedPoint
+
+from agent0.hyperdrive.interactive import Chain, InteractiveHyperdrive, LocalChain
+from agent0.hyperdrive.interactive.event_types import OpenLong, OpenShort
+from agent0.hyperdrive.interactive.interactive_hyperdrive_agent import InteractiveHyperdriveAgent
+from agent0.hyperdrive.state.hyperdrive_actions import HyperdriveActionType
+
+# %%
+# Variables by themselves print out dataframes in a nice format in interactive mode
+# pylint: disable=pointless-statement
+
+
+# TODO: change this into an executable script with LOCAL=False always once we're sure it is working
```

Comment:
We've decided not to allow non-local runs here. So you can delete lines 18 & 19, and then the conditionals on lines 24-31 can be changed to only include lines 25-28.
```python
NUM_TRADES = 3
NUM_PATHS_CHECKED = 10

# Parameters for local chain initialization, defines defaults in constructor
# set a large block time so i can manually control when it ticks
chain_config = LocalChain.Config(block_time=1_000_000)
chain = LocalChain(config=chain_config)
```